types/react-query/index.d.ts

```diff
@@ -1,29 +1,46 @@
-// Type definitions for react-query 0.3
+// Type definitions for react-query 1.0
 // Project: https://github.com/tannerlinsley/react-query
 // Definitions by: Lukasz Fiszer <https://github.com/lukaszfiszer>
 //                 Jace Hensley <https://github.com/jacehensley>
 //                 Matteo Frana <https://github.com/matteofrana>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
+// Minimum TypeScript Version: 3.0
 import { ComponentType } from 'react';
 
-// overloaded useQuery function with pagination
-export function useQuery<TResult, TVariables extends object>(
-    queryKey: QueryKey<TVariables>,
-    queryFn: QueryFunction<TResult, TVariables>,
-    options: QueryOptionsPaginated<TResult>
-): QueryResultPaginated<TResult, TVariables>;
+export type QueryKey = string | [string, ...any[]] | false | null;
```

Comment:
I would add ``undefined`` to this union.

What do you think ?

Reply:
ok